src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value, result) {
```

Comment:
Need to use more appropriate variable names.
What you accept as an argument - is not a result.
It can be `number1` and `number2` or `firstNumber` and `secondNumber`